regolith/builders/readinglistsbuilder.py

```diff
@@ -0,0 +1,123 @@
+"""Builder for Reading Lists."""
+
+from copy import deepcopy, copy
+import datetime, sys
+import requests
+from habanero import Crossref
+
+from regolith.builders.basebuilder import LatexBuilderBase
+from regolith.fsclient import _id_key
+from regolith.sorters import position_key
+from regolith.tools import (
+    all_docs_from_collection,
+    fuzzy_retrieval,
+    number_suffix,
+)
+from regolith.stylers import sentencecase, month_fullnames
+from regolith.dates import month_to_int
+
+
+class ReadingListsBuilder(LatexBuilderBase):
+    """Build reading lists from database entries"""
+
+    btype = "reading-lists"
+    needed_dbs = ['people', 'reading_lists']
+
+    def construct_global_ctx(self):
+        """Constructs the global context"""
+        super().construct_global_ctx()
+        gtx = self.gtx
+        rc = self.rc
+        self.cr = Crossref()
+        cr = self.cr
+        gtx["people"] = sorted(
+            all_docs_from_collection(rc.client, "people"),
+            key=position_key,
+            reverse=True,
+        )
+        gtx["reading_lists"] = sorted(
+            all_docs_from_collection(rc.client, "reading_lists"), key=_id_key
+        )
+        gtx["all_docs_from_collection"] = all_docs_from_collection
+        gtx["float"] = float
+        gtx["str"] = str
+        gtx["zip"] = zip
+
+    def latex(self):
+        """Render latex template"""
+
+        for rlist in self.gtx["reading_lists"]:
+            listid = rlist["_id"]
+            outfile_bib = listid
+
+            n = 1
+            for paper in rlist['papers']:
+                # fixme: code here to get info from crossref
```

Comment:
not sure what information should be got from crossref...

Reply:
We get the citation info from crossref.  But that should already be in there.  I will look at your whole file changes to see what is going on.

Reply:
OK, I see.  The code is below (starting ``article = self.cr.works(ids=doi)``

We can just delete this comment.  Also the commented out code below with the ``pass`` can be deleted.